src/cookbooks/additional_props.md

```diff
@@ -0,0 +1,50 @@
+# Additional Properties
+
+If some field is present in a valid received [ping](../concepts/terminology.md#ping),
+but is not present in the ping's [schema](../concepts/terminology.md#schema),
+it doesn't have its own column to be placed into during
+[ingestion](../concepts/terminology.md#ingestion).
+Instead, those fields remain as raw JSON and are placed in the
+`additional_properties` column of the ping's table or view.
+
+This can happen for a variety of usually-temporary problems like:
+
+- The latest schema hasn't yet been deployed (see ["What does it mean when a schema deploy is blocked?"](../concepts/pipeline/schemas.html#what-does-it-mean-when-a-schema-deploy-is-blocked))
+- [mozilla-pipeline-schemas](https://github.com/mozilla-services/mozilla-pipeline-schemas) was not updated after a change in the data which was submitted (this shouldn't happen with [Glean](../concepts/glean.md), but can happen with some legacy data, for example the legacy desktop telemetry envionment).
```

Comment:
Thanks for addressing this! It also occurs to me that we could link to a reference on what the telemetry environment is.

```suggestion
- [mozilla-pipeline-schemas](https://github.com/mozilla-services/mozilla-pipeline-schemas) was not updated after a change in the data which was submitted (this shouldn't happen with [Glean](../concepts/glean.md), but can happen with some legacy data, for example the [legacy desktop telemetry environment](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/environment.html)).
```

Reply:
Can't apply outdated suggestions : |